### PR TITLE
deprecate gazetteer implementation

### PR DESCRIPTION
**Pull request by warm-coolguy**
## Summary

The gazetteer implementation has been marked as deprecated for the next major release.

## Instructions for local reproduction and review

No relevant changes.

### CODE REVIEW

File: packages/plugins/AddressSearch/src/utils/searchMethods/getSearchMethod.ts [hunk 1]
```diff
@@ -21,6 +21,11 @@ export const getMethodContainer = () => {
     })
 
   function getSearchMethod(type: string): SearchMethodFunction {
+    if (type === 'gazetteer') {
+      console.warn(
+        'Search type "gazetteer" is deprecated. Please use "mpapi" instead.'
```
Comment (dopenguin) [inline packages/plugins/AddressSearch/src/utils/searchMethods/getSearchMethod.ts:26]:
```suggestion
        '@polar/plugin-address-search: Search type "gazetteer" is deprecated. Please use "mpapi" instead.'
```

**Review by dopenguin — CHANGES_REQUESTED** [review]
🏓 @warm-coolguy

**Comment by warm-coolguy** [comment]
> 🏓 @warm-coolguy

🏓 @dopenguin

**Review by dopenguin — APPROVED** [review]
🚀 @warm-coolguy

**Pull request merged**